Jenkinsfile

```diff
@@ -827,6 +827,11 @@ def runConjurTests(run_only_str) {
       "Policy Parser - ${env.STAGE_NAME}": {
         sh 'cd gems/policy-parser && ./test.sh'
       }
+    ],
+    "conjur_rack": [
+      "Rack - ${env.STAGE_NAME}": {
+        sh 'cd gems/conjur-rack && ./test.sh'
```

Comment:
It looks like the reporting of test results needs to also be ported to this Jenkinsfile:
https://github.com/cyberark/conjur-rack/blob/master/Jenkinsfile#LL18C1-L18C35